<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GH-CZQVD63" ignored="0" name="Test" tests="20" failures="7" timestamp="2021-04-19T11:35:36 IST" time="39.096" errors="0">
  <testcase name="registration" time="23.659" classname="com.api.tests.RegistrationTest"/>
  <testcase name="registrationWithInvalidEmail" time="0.846" classname="com.api.tests.RegistrationTest"/>
  <testcase name="registrationWithMissingEmailAndPassword" time="0.799" classname="com.api.tests.RegistrationTest"/>
  <testcase name="registrationWithMissingPassword" time="0.814" classname="com.api.tests.RegistrationTest"/>
  <testcase name="loginIfEmailAndPasswordMissing" time="0.79" classname="com.api.tests.LoginTest"/>
  <testcase name="loginIfEmailIdInvalid" time="0.748" classname="com.api.tests.LoginTest"/>
  <testcase name="verifyFailedLogin" time="0.74" classname="com.api.tests.LoginTest"/>
  <testcase name="verifyIfIncorrectPassword" time="0.776" classname="com.api.tests.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at com.api.tests.LoginTest.verifyIfIncorrectPassword(LoginTest.java:94)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verifyIfIncorrectPassword -->
  <testcase name="verifySuccessfulLogin" time="0.714" classname="com.api.tests.LoginTest"/>
  <testcase name="createSuccess" time="0.771" classname="com.api.tests.CreateTest"/>
  <testcase name="createWithMissingJob" time="0.74" classname="com.api.tests.CreateTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.api.tests.CreateTest.createWithMissingJob(CreateTest.java:68)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createWithMissingJob -->
  <testcase name="createWithMissingName" time="0.749" classname="com.api.tests.CreateTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.api.tests.CreateTest.createWithMissingName(CreateTest.java:55)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createWithMissingName -->
  <testcase name="createWithMissingNameAndJob" time="1.806" classname="com.api.tests.CreateTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.api.tests.CreateTest.createWithMissingNameAndJob(CreateTest.java:42)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createWithMissingNameAndJob -->
  <testcase name="deleteSuccessful" time="0.755" classname="com.api.tests.DeleteUserTest"/>
  <testcase name="deleteWithInvalidFormatParam" time="0.703" classname="com.api.tests.DeleteUserTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [204]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [204]
at com.api.tests.DeleteUserTest.deleteWithInvalidFormatParam(DeleteUserTest.java:46)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- deleteWithInvalidFormatParam -->
  <testcase name="deleteWithInvalidIdParam" time="0.804" classname="com.api.tests.DeleteUserTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [204]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [204]
at com.api.tests.DeleteUserTest.deleteWithInvalidIdParam(DeleteUserTest.java:35)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- deleteWithInvalidIdParam -->
  <testcase name="deleteWithMissingIdParam" time="0.713" classname="com.api.tests.DeleteUserTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [204]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [204]
at com.api.tests.DeleteUserTest.deleteWithMissingIdParam(DeleteUserTest.java:24)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- deleteWithMissingIdParam -->
  <testcase name="verifyGetSingleUser" time="0.333" classname="com.api.tests.GetUserListTest"/>
  <testcase name="verifyGetSingleUserNotFound" time="0.333" classname="com.api.tests.GetUserListTest"/>
  <testcase name="verifyGetUserList" time="0.957" classname="com.api.tests.GetUserListTest"/>
</testsuite> <!-- Test -->
